cypress/integration/checkin/checkin.test.spec.js

```diff
@@ -43,7 +45,9 @@ describe('Check In - Actions ', () => {
   });
 
   it('C347631 Check in: Basic check in', { tags: [TestTypes.smoke] }, () => {
-    CheckInActions.checkInItem();
+    cy.visit(TopMenu.checkInPath);
```

Comment:
provide path to login method instead of extra navigation into the test

Reply:
Done